include/seqan3/range/view/get.hpp

```diff
@@ -0,0 +1,101 @@
+                        See below for the properties of the returned range.
+ * \ingroup view
+ *
+ * ### View properties
+ *
+ * | range concepts and reference_t  | `urng_t` (underlying range type)      | `rrng_t` (returned range type)                          |
+ * |---------------------------------|:-------------------------------------:|:-------------------------------------------------------:|
+ * | std::ranges::InputRange         | *required*                            | *preserved*                                             |
+ * | std::ranges::ForwardRange       |                                       | *preserved*                                             |
+ * | std::ranges::BidirectionalRange |                                       | *preserved*                                             |
+ * | std::ranges::RandomAccessRange  |                                       | *preserved*                                             |
+ * | std::ranges::ContiguousRange    |                                       | *lost*                                                  |
+ * |                                 |                                       |                                                         |
+ * | std::ranges::ViewableRange      | *required*                            | *preserved*                                             |
+ * | std::ranges::View               |                                       | *preserved*                                             |
+ * | std::ranges::SizedRange         |                                       | *preserved*                                             |
+ * | std::ranges::CommonRange        |                                       | *preserved*                                             |
+ * | std::ranges::OutputRange        |                                       | *preserved*                                             |
+ * | seqan3::const_iterable_concept  |                                       | *preserved*                                             |
+ * |                                 |                                       |                                                         |
+ * | seqan3::reference_t             | seqan3::tuple_like_concept            | std::tuple_element_t<index, seqan3::reference_t<urng_t>>|
+ *
+ * See the \link view view submodule documentation \endlink for detailed descriptions of the view properties.
+ *
+ * ### Example
+ *
+ * \snippet test/snippet/range/view/get.cpp usage
+ * \hideinitializer
+ */
+template <size_t index>
+inline auto const get = view::transform([] (auto && in)
+    -> std::conditional_t<std::is_lvalue_reference_v<decltype(in)> && !std::is_const_v<decltype(in)>,
+                          std::tuple_element_t<index, remove_cvref_t<decltype(in)>> &,
+                          std::tuple_element_t<index, remove_cvref_t<decltype(in)>>>
+{
```

Comment:
Sorry, last comment, I checked this again and the correct approach is just to write:
```cpp
inline auto const get = view::transform([] (auto && in) -> auto &&
{
```
It also covers the `const`-cases and is much simpler to read.

Reply:
Done! Also rebased on master again.